lifecycle/lifecycle/event_stream/server.py

```diff
@@ -1,8 +1,15 @@
 import asyncio
 import threading
-
-import socketio
 import time
+import warnings
+
+warnings.filterwarnings(
```

Comment:
This whole thing seems a tad cryptic, why are we ignoring a deprecation warning? Should we not update `falcon.media` if it's deprecated? If the answer is we shouldn't, then maybe adding a comment explaining why would be prudent.

Reply:
There's an issue which is still open https://github.com/falconry/falcon/issues/2066